src/test/ui/consts/const-eval/assign-to-static-within-other-static-2.rs

```diff
@@ -1,7 +1,7 @@
 // New test for #53818: modifying static memory at compile-time is not allowed.
 // The test should never compile successfully
 
-#![feature(const_raw_ptr_deref, const_mut_refs)]
+#![feature(const_mut_refs)]
```

Comment:
this test is now equal to src/test/ui/consts/const-eval/mod-static-with-const-fn.rs, so you can remove one of them